Mathlib/Data/List/Defs.lean

```diff
@@ -17,11 +17,22 @@ proofs about these definitions, those are contained in other files in `Mathlib.D
 namespace List
 
 /-- Split a list at an index.
-     splitAt 2 [a, b, c] = ([a, b], [c]) -/
+
+    splitAt 2 [a, b, c] = ([a, b], [c]) -/
 def splitAt : ℕ → List α → List α × List α
 | n+1, x :: xs => let (l, r) := splitAt n xs; (x :: l, r)
 | _, xs => ([], xs)
 
+/-- Split a list at an index. Ensures the left list always has the specified length
+by right padding with the provided default element.
+
+    splitAtD 2 [a, b, c] x = ([a, b], [c])
```

Comment:
I'm not sure if we should rely on indentation for code blocks...  But that's a bigger discussion better had elsewhere.